### DISCUSSION TITLE
Intellisense with the C# Dev Kit?

### DISCUSSION THREAD

**Discussion by bwong365** [cake-build/cake#4389] [Q&A] [answered]
Are there any plans to get Cake intellisense working with the C# Dev Kit? I noticed that the only way I could get intellisense to work was to check "Use Omnisharp" in VSCode, but as it notes that the option won't be honored with the C# Dev Kit installed; I had to temporarily uninstall it just to get intellisense, which isn't great DX.

**Comment by devlead** [comment, marked as answer]
Unfortunately C# DevKit has gone partially closed source, and doesn't take contributions and at the moment doesn't offer any supported extension points for third parties.

We've provided feedback to Microsoft, but it unfortunately doesn't seem to be an priority,

There's an issue about public language server API 

https://github.com/dotnet/roslyn/issues/68696
